### PR TITLE
Update header bar styling for GB and EV

### PR DESCRIPTION
**Pull request by imransl**
## Description
Update the header bar styling in Genome browser and Entity viewer

## Related JIRA Issue(s)
[ENSWBSITES-2445](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2445)

## Deployment URL(s)
http://update-headerbar-styling.review.ensembl.org

## Views affected
Genome browser and Entity viewer

### CODE REVIEW

**Comment by azangru** [comment]
This is from the review deployment. The main bug seems to be still present.

image

File: src/shared/components/feature-summary-strip/VariantSummaryStrip.tsx [hunk 1]
```diff
@@ -70,28 +77,36 @@ const VariantSummaryStrip = (
...
-        <span className={styles.featureNameEmphasized}>{variant.name}</span>
+      <div className={featureStripStyles.section}>
+        <span className={featureStripStyles.featureSummaryStripLabel}>
+          Variant
+        </span>
+        <span className={featureStripStyles.featureNameEmphasized}>
+          {variant.name}
+        </span>
+        <span className={styles.variantType}>
+          {referenceAllele?.allele_type.value}
```
Comment (azangru) [inline src/shared/components/feature-summary-strip/VariantSummaryStrip.tsx:95]: If you do this, then the label will always read "biological_region" (this is what the type of the reference allele always is).

What you should use instead is `variant->allele_type->value`

File: src/content/app/entity-viewer/variant-view/variant-summary-strip/VariantSummaryStrip.module.css [hunk 2]
```diff
@@ -24,10 +24,10 @@
 
 .emphasized {
   font-weight: var(--font-weight-bold);
+  margin-right: 12px;
```
Comment (azangru) [inline src/content/app/entity-viewer/variant-view/variant-summary-strip/VariantSummaryStrip.module.css:27]: This contradicts the name of the class. And also, you no longer seem to be using this class anywhere; you are using `featureStripStyles.featureNameEmphasized` in `VariantSummaryStrip` component

Comment (imransl) [reply src/content/app/entity-viewer/variant-view/variant-summary-strip/VariantSummaryStrip.module.css:27]: It's probably the other `VariantSummaryStrip` component that you are talking about. This class is used in `src/content/app/entity-viewer/variant-view/variant-summary-strip/VariantSummaryStrip.tsx`.

**Review by azangru — APPROVED** [review]

**Pull request merged**